src/pages/registration/contributors_tab/components/AuthorCard.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  TableRow,
+  TableCell,
+  TextField,
+  Typography,
+  Tooltip,
+  FormControlLabel,
+  Checkbox,
+  Button,
+} from '@material-ui/core';
+import { FormikProps, useFormikContext, FieldProps, Field } from 'formik';
+import React, { FC, useState, ChangeEvent } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Contributor, UnverifiedContributor } from '../../../../types/contributor.types';
+import { ContributorFieldNames, SpecificContributorFieldNames } from '../../../../types/publicationFieldNames';
+import { Registration } from '../../../../types/registration.types';
+import AffiliationsCell from './AffiliationsCell';
+import WarningIcon from '@material-ui/icons/Warning';
+import CheckIcon from '@material-ui/icons/Check';
+import DeleteIcon from '@material-ui/icons/Delete';
+import styled from 'styled-components';
+
+const StyledWarningIcon = styled(WarningIcon)`
+  color: ${({ theme }) => theme.palette.warning.main};
+`;
+
+const StyledCheckIcon = styled(CheckIcon)`
+  color: ${({ theme }) => theme.palette.success.main};
+`;
+
+const StyledEmailTextField = styled(TextField)`
+  margin-bottom: 0.5rem;
+`;
+
+interface AuthorCardProps {
+  author: Contributor;
+  onMoveAuthor: (event: React.ChangeEvent<any>) => void;
+  onRemoveAuthorClick: () => void;
+  setUnverifiedAuthor: (unverifiedAuthor: UnverifiedContributor) => void;
+}
+
+const AuthorCard: FC<AuthorCardProps> = ({
+  author,
+  onMoveAuthor: onMoveCard,
+  onRemoveAuthorClick,
+  setUnverifiedAuthor,
+}) => {
+  const { t } = useTranslation('registration');
+  const index = author.sequence - 1;
+  const baseFieldName = `${ContributorFieldNames.CONTRIBUTORS}[${index}]`;
+  const { values, setFieldValue }: FormikProps<Registration> = useFormikContext();
```

Comment:
```suggestion
  const { values, setFieldValue } = useFormikContext<Registration>();
```
fant nylig ut at dette var det samme, da slipper vi unna en ekstra import

Reply:
uhh, nice 👍